pages/Getting Started/Installation.md

```diff
@@ -70,6 +70,12 @@ or a tagged release from the arch packages:
 sudo pacman -S hyprland
 ```
 
+Alternatively, install the `hyprland-meta` package to automatically fetch and compile the latest git versions of all components within the Hypr ecosystem.
```

Comment:
it's hypr* ecosystem not hypr ecosystem xd

Reply:
Hmmm https://wiki.hyprland.org/Hypr-Ecosystem/

Reply:
I fear we can't put * in titles without something breaking